### DISCUSSION TITLE
Built-in and custom fields work simultaneously

### DISCUSSION THREAD

**Discussion by GoldGem** [clientIO/joint#2610] [Q&A]
### Introduction

I want to add custom fields to inspector via renderFieldContent function. But after adding this function, built-in fields (select, text, textarea etc.) are no longer displayed, and only custom fields are displayed. How to make built-in and custom fields work simultaneously?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
It was a bug in `v4.0.0`. It's been fixed in `v4.0.1`.
